### PR TITLE
Oss add audit deployment

### PR DESCRIPTION
**Pull request by aloncarmel111**
What does this PR do?
Update changelog, readme and audit.md files with the current audit design

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -5,6 +5,10 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
+### Added
```
Comment (uCatu) [inline CHANGELOG.md]: Version bump is required to 1.5.0
@jvanderhoof WDYT

Comment (sashaCher) [reply CHANGELOG.md]: We haven't added functionality from end user point of view. No APIs have been changed/added so I think that it should be 1.4.3 rather 1.5.0.

Comment (aloncarmel111) [reply CHANGELOG.md]: As discussed with @garkler, we need to add our changelog to the unreleased version in first phase.

**Comment by uCatu** [comment]
Reverted Audit US.

**Pull request closed without merging**